linux/network_device_ops/pvrdma_network_device_ops.yml

```diff
@@ -120,19 +143,25 @@
           ignore_errors: true
           register: rping_server_result
 
-        - name: "Display RDMA ping result"
+        - name: "Display RDMA ping result from client VM to server VM"
           ansible.builtin.debug: var=rping_server_result
 
-        - name: "Check the result of RDMA pinging server VM from client VM"
+        - name: "Check RDMA ping result from client VM to server VM"
           ansible.builtin.assert:
             that:
               - rping_server_result is defined
               - rping_server_result.rc is defined
               - rping_server_result.rc == 0
             fail_msg: >-
-              RDMA pinging server VM from client VM failed.
+              Failed to run RDMA ping from client VM '{{ pvrdma_client_vm_name }}'
+              to server VM '{{ pvrdma_server_vm_name }}' failed.
               Hit error '{{ rping_server_result.stderr | default() }}'
 
+        # Switch to server VM
```

Comment:
Why not put the task to line 176 where start the ops on server side?

Reply:
To avoid misunderstanding that server VM is removed.